packages/manager/apps/web/client/app/app.js

```diff
@@ -43,6 +43,7 @@ import orderContracts from './components/manager-order-contracts';
 import orderPaymentMeans from './components/manager-order-payment-means';
 import orderService from './components/manager-order-service/manager-order-service.service';
 
+import domain from './domain';
 import domainEmailObfuscation from './domain/email-obfuscation/index';
```

Comment:
Not yours but it seems that those are submodules belonging to `domain`, would it be possible to import them in the `domainModule` instead ?